tests/end_to_end/test_helper.cpp

```diff
@@ -219,8 +219,7 @@ _get_environment_variable_as_bool(const std::string& name)
     return true;
 }
 
-HANDLE
-GlueCreateFileW(
+_Success_(return != INVALID_HANDLE_VALUE) HANDLE GlueCreateFileW(
```

Comment:
I don't see any value in this change.  My understanding of `_Success_` (from https://learn.microsoft.com/en-us/cpp/code-quality/understanding-sal?view=msvc-170) is that it is only meaningful when you have an `_Out_` argument, in which case it's a statement about whether its value is meaningful on return.  I guess it's not harmful to add this but I don't see it as helpful.